src/binaries/BinariesUtilityV2.ts

```diff
@@ -72,39 +68,6 @@ export class BinariesUtilityV2 implements IBinariesUtility {
         return this._downloadProgress;
     }
 
-    /**
```

Comment:
```
   /**
      * If the binaries aren't downloaded, download newest versions.
      * If the binaries is already downloaded, check the version the first time.
      * If the version is insufficient, then download the newest version.
      * If the version is sufficient, then all good!
      * Returns the BridgeClient, KubectlClient to enable working with the clients.
      * Does not retry.
      */ 
```

This functionality seems like we can use it for the cases of say new version of `b2k` gets released, how will we make sure that extension downloads that version?